linux-arm-cortex/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM dockcross/base:latest
+MAINTAINER Declan Zhang <deqzhang@ucdavis.edu>
+
+# Download and install cross-compiler binaries
+RUN wget https://releases.linaro.org/components/toolchain/binaries/latest/arm-eabi/gcc-linaro-6.3.1-2017.02-x86_64_arm-eabi.tar.xz
+RUN wget https://releases.linaro.org/components/toolchain/binaries/latest/aarch64-elf/gcc-linaro-6.3.1-2017.02-x86_64_aarch64-elf.tar.xz
```

Comment:
Please remove this and possibly  include it in another dockcross image -- we have one toolchain per image. This toolchain could be called `dockcross/bare-arm64` to be consistent with the other image names.